en/assembly/quick_start_cuav_v5_plus.md

```diff
@@ -72,17 +75,22 @@ If using a Spektrum satellite receiver, this should be connected to the flight c
 
 ## Power
 
-The V5+ kit includes the HV\_PM, which supports 2~10s LiPo battery.
-Please connect the 6pin connector of the HW\_PM module to the flight control `Power1` interface.
+The V5+ kit includes the *HV\_PM* module, which supports 2~10S LiPo batteries.
+Connect the 6pin connector of the *HW\_PM* module to the flight control `Power1` interface.
+
+> **Warning** The supplied power module is unfused (see [this issue](../flight_controller/cuav_v5_plus.md#issue_pm_unfused)).
 
 ![V5+ AutoPilot](../../assets/flight_controller/cuav_v5_plus/connection/v5+_quickstart_01.png)
 
+> **Note** The power module does not supply the output rail.
+  If you're connecting servos/actuators directly of this rail you will need to separately power it using a BEC.
+
```

Comment:
@cuhome As discussed, added a note about powering the output rail. Please check.

For this board it is more likely it will be used for VTOLs/FW and things that use servos powered off the servo rail. Would be much better if this had a diagram. Could you supply?

Also, this case supposedly supports an I2C battery. Can we explain where that might be obtained and how it is connected?

Reply:
Sorry, I don't understand the output rail you said, I may need jinger to help me.

Reply:
@cuhome 

> Sorry, I don't understand the output rail you said, I may need jinger to help me.

I changed terminology a bit to make this clear. But basically on an old 3DR Pixhawk if you wanted to attach a servo to the PWM outputs you would need to connect power to the outputs - this was referred to as the output rail.

What I am asking though, is how would I wire up a servo (say), and can I please have an image. My assumption is that if you wire up one servo with a BEC, all the other PWM outputs are also powered.

Reply:
... and I now see this got answered by David too above. Thanks.